fenix/app/src/main/res/values/strings.xml

```diff
@@ -1302,7 +1302,9 @@
     <!-- Title text of a delete browsing data dialog. -->
     <string name="delete_history_prompt_title">Time range to delete</string>
     <!-- Body text of a delete browsing data dialog. -->
-    <string name="delete_history_prompt_body">Removes history (including history synced from other devices), cookies and other browsing data.</string>
```

Comment:
please add `moz:RemovedIn=` for `delete_history_prompt_body` similar to [this](https://github.com/mozilla-mobile/firefox-android/blob/c08cbd463a9a2b82ae094a1b1d1ce480693c4412/fenix/app/src/main/res/values/strings.xml#L396).